openedx/core/djangoapps/ace_common/message_types.py

```diff
@@ -0,0 +1,23 @@
+"""
+Subclasses for MessageType that are localized to Open edX platform.
+"""
+import attr
+
+from django.conf import settings
+
+from edx_ace.message import MessageType
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+
+
+@attr.s(cmp=False)
+class SitesAwareMessageType(MessageType):
```

Comment:
@mulby that's aboslutely right. It turns out overriding the `__init__` method is the best way to do such overriding. Redefining `options` was very problematic at best. Since `attr.s` classes don't support both inheritance and overriding at the same time in an easy way, `__init__` override seemed like the best option.